newsfragments/43902.significant.rst

```diff
@@ -0,0 +1,5 @@
+Renamed ``execution_date`` to ``logical_date`` across the codebase to align with Airflow 3.0.
+
+The shift towards ``logical_date`` helps move away from the limitations of ``execution_date``, particularly with dynamic DAG runs and cases where multiple runs occur at the same time. This change impacts database models, templates, and functions:
```

Comment:
```suggestion
The shift towards ``logical_date`` helps move away from the limitations of ``execution_date``, particularly with dynamic DAG runs and cases where multiple runs occur at the same time. This change impacts database models, templates, and functions:

```

Needs an extra empty line to fix rendering.

Reply:
Changed it